Formula/rust.rb

```diff
@@ -79,6 +85,21 @@ def install
       system "make", "install"
     end
 
+    if build.with? "racer"
+      resource("racer").stage do
+        ENV.append_path "PATH", bin
+        cargo_home = buildpath/"cargo_home"
+        cargo_home.mkpath
+        ENV["CARGO_HOME"] = cargo_home
+        system bin/"cargo", "build", "--release", "--verbose"
+        (libexec/"bin").install "target/release/racer"
+        (bin/"racer").write_env_script(libexec/"bin/racer", :RUST_SRC_PATH => pkgshare/"rust_src")
+      end
+      # remove any binary files as homebrew will run ranlib on them and barf, keep everything else
```

Comment:
please wrap the comment at the eightieth column, capitalize Homebrew, and avoid the run-on sentence